src/main/resources/static/css/forms.css

```diff
@@ -18,3 +18,7 @@
 .logout-form {
     background-color: transparent;
 }
+
+.edit-form {
```

Comment:
would prefer a more common class to use here instead of <page>-form, e.g. form-mt (=form with margin top)